aries_cloudagent/config/argparse.py

```diff
@@ -1638,6 +1638,17 @@ def add_arguments(self, parser: ArgumentParser):
                 " is expected."
             ),
         )
+        parser.add_argument(
+            "--wallet-rekey-derivation-method",
+            type=str,
+            metavar="<rekey-derivation-method>",
+            env_var="ACAPY_WALLET_REKEY_DERIVATION_METHOD",
+            help=(
+                "Specifies the key derivation method used for the replacement"
+                "rekey encryption. If RAW key derivation method is used, also"
+                "--wallet-key parameter is expected."
```

Comment:
I think it should be --wallet-rekey instead of --wallet-key

Reply:
Yes. I copied and pasted from wallet-key and didn't go over it properly.